docs/source/tracking.rst

```diff
@@ -180,6 +178,77 @@ and the `LocalArtifactRepository <https://github.com/mlflow/mlflow/blob/master/m
 `S3ArtifactRepository <https://github.com/mlflow/mlflow/blob/master/mlflow/store/artifact/s3_artifact_repo.py#L14>`_ are
 concrete implementations of the abstract class `ArtifactRepository <https://github.com/mlflow/mlflow/blob/master/mlflow/store/artifact/artifact_repo.py#L13>`_.
 
+Scenario 5: MLflow Tracking Server enabled as proxy for artifact storage access
+-------------------------------------------------------------------------------
+
+MLflow's Tracking Server supports utilizing the host as a proxy server for operations involving artifacts.
+Once configured with the appropriate access requirements, an administrator can start the tracking server to enable assumed-role operations involving the saving, loading, or listing of
+model artifacts, images, documents, and files. This eliminates the need to allow end users to have direct path access to a remote file store for artifact handling and eliminates the
+need for an end-user to provide access credentials to interact with an underlying file store.
+
+.. figure:: _static/images/scenario_5.png
+
+Enabling the Tracking Server to serve as an artifact proxy routes artifact operation requests and file transfers through the tracking server:
+
+ * **Part 1a and b**:
+
+  * The MLflow client creates an instance of a `RestStore` and sends REST API requests to log MLflow entities
+  * The Tracking Server creates an instance of an `SQLAlchemyStore` and connects to the remote host for inserting
+    tracking information in the database (i.e., metrics, parameters, tags, etc.)
+
+ * **Part 1c and d**:
+
+  * Retrieval requests by the client return information from the configured `SQLAlchemyStore` table
+
+ * **Part 2a and b**:
+
+  * Logging events for artifacts are made to the ``mlflow-artifacts:/`` uri for saving
+  * The Tracking server, serving as proxy, interfaces with the configured backend store, using its configured authorization to interact with the file store for writing artifacts
```

Comment:
Specifically, the client uses `HttpArtifactRepository` (https://github.com/mlflow/mlflow/blob/3c434246f73b41835194f3e6aa1f9be8488bcd56/mlflow/store/artifact/http_artifact_repo.py#L11) to upload and download files to / from the remote host where the MLflow Tracking server is running.

Reply:
fixed the wording here.